changelog/789.feature.rst

```diff
@@ -0,0 +1 @@
+Increase the upper bound for the ``aiohttp`` dependency from ``<3.9`` to ``<4``.
```

Comment:
Should probably also be labeled as `misc` instead of `feature`, it's only a dependency bound update.